storage/rocksdb/rdb_utils.h

```diff
@@ -473,4 +473,13 @@ class Ensure_initialized {
   return fn.compare(fn_len - ext_len, ext_len, extension) == 0;
 }
 
+// return <db_name>.<table_name>
+inline std::string get_full_tablename(const char *const db_name,
+                                      const char *const table_name) {
+  std::string fullname = db_name;
+  fullname.append(".");
```

Comment:
Nit: since this is a single character, a single-character version should be more efficient:
```suggestion
fullname += '.';
```

Reply:
Good point.. just realized it can be value type..

 https://github.com/llvm-mirror/libcxx/blob/master/include/string#L985